common/azure-common-test/src/main/java/com/azure/common/test/namer/ResourceNamerFactory.java

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.common.test.namer;
+
+/**
+ * The ResourceNamerFactory to generate ResourceNamer.
+ */
+public class ResourceNamerFactory {
+
+    /**
+     * Factory method to generate instance of ResourceNamer.
+     * @param name prefix for the names.
+     * @return instance of ResourceNamer
+     */
+    public ResourceNamer createResourceNamer(String name) {
+        return new ResourceNamer(name);
+    }
+}
```

Comment:
Why do we need a factory for this - it doesn't seem necessary?

Reply:
Some of the sdk client tests create Namer object through factory. If not using this, might change a lot of codes in sdk client tests.